### PR TITLE
Updated the Readme

### LINKED ISSUE #1: Edit README.md
**Is your feature request related to a problem? Please describe.**
Edit README.md to reflect the [2FA design doc](https://bit.ly/3AcD8m7)

**To-add**
- [ ] Project Overview (Shortened)
- [ ] Links
  - [ ]  design document
  - [ ] outside resources listed in Bibliography

**To-remove**
- [ ] Layout/readme
- [ ] Issues
- [ ] git specific stuff/linux and everything underneath
- [ ] conclusion

### PR DESCRIPTION
**Pull request by alans-42**
Updated the readme to include information from the wiki.
closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,97 +1,54 @@
...
-If you have already created a branch or forked the repo, and are logged in, github should automagically detect that and prompt you to start the process!
-
-There is a template to follow, which should already be there for you to follow, describing your changes and what they do.
-After this, you can submit your pull request and wait for feedback!
-
-## conclusion
-
-This readme will be constantly changing and is very open to feedback and clarification through issues.
+## Contact
+Emil Kovacev Junior Computer Science BS – University at Buffalo
```
Comment (emilkovacev) [inline README.md]: This can probably be deleted

File: README.md [hunk 2]
```diff
@@ -1,97 +1,54 @@
-# Project Template Repository
+# Authenticator
 
-This repository is designed to be a template for how to layout your projects for MakeOpenSource.
+An authenticator app is an application that acts as a secondary device to verify your identity when logging in to account-based platforms as another level of security. 2-factor authenticator apps utilize another device as an additional key to unlock accounts. An open-source authenticator app with additional quality-of-life features that easily integrates into existing applications and offers similar/better levels of security and convenience as paid alternatives.
```
Comment (emilkovacev) [inline README.md]: There's a grammatical error in the third sentence, should start with something along the lines of "MakeOpenSource Authenticator is an open-source authenticator app with..." (I know this is from the wiki **I wrote** but I just noticed it so we may as well change it here 😄)

**Comment by emilkovacev** [comment]
Other than those small comments (one of which was **my mistake**) this pull request looks ready to merge. Great job!

**Comment by emilkovacev** [comment]
To clarify your next steps:
- If you make commits to your forked branch, they will be added to the pull request here
- Before I can merge this pull request, please make commits that resolve the above messages in the review!

**Comment by alans-42** [comment]
Where would I find these comments?

**Comment by emilkovacev** [comment]
Click on the "files changed" tab right above the first comment in this thread, and it will show the 2 suggestions I made

**Review by emilkovacev — APPROVED** [review]

**Pull request merged**